components/calendar/calendar.js

```diff
@@ -572,11 +636,23 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 	_updateShownMonthDecrease() {
 		if (this._shownMonth === 0) this._shownYear--;
 		this._shownMonth = getPrevMonth(this._shownMonth);
+		this.classList.remove('d2l-calendar-initial-month');
```

Comment:
What do you think of adding/removing these classes from an internal `div` rather than the host element?  It would enable them to be applied in `render` where we usually want DOM mutations to be done.  Alternatively, when I've needed to expose a "private" bit of state on the host, I've defined a private property that gets reflected, which makes it clear that it's private, where as `class` on the host is very much a "public" property.

Reply:
I think putting it on an internal div is the "cleanest" as it hides everything, then yeah the private reflected property would be next-best.

Reply:
Yup that does sound cleaner - I'll give that a try.